spec/stat_calculator_spec.rb

```diff
@@ -90,14 +90,28 @@
   end
 
   describe '#best_offense' do
-    it "Returns name of the team with the highest average number of goals scored per game across all seasons" do
+    it "returns name of the team with the highest average number of goals scored per game across all seasons" do
       expect(@stat_calculator.best_offense).to eq("Reign FC")
     end
   end
 
   describe '#worst_offense' do
-    it "Returns name of the team with the lowest average number of goals scored per game across all seasons" do
+    it "returns name of the team with the lowest average number of goals scored per game across all seasons" do
       expect(@stat_calculator.worst_offense).to eq("Utah Royals FC")
     end
   end
+
```

Comment:
Updates Tests
The existing tests for most_tackles and fewest_tackles are sufficient, but you could add edge cases:
```ruby
describe '#most_tackles' do
  it 'returns the name of the team with the most tackles for a specific season' do
    expect(@stat_calculator.most_tackles("20132014")).to eq("FC Cincinnati")
    expect(@stat_calculator.most_tackles("20142015")).to eq("Seattle Sounders FC")
  end

  it 'returns nil if there are no games for the season' do
    expect(@stat_calculator.most_tackles("99999999")).to be_nil
  end
end

describe '#fewest_tackles' do
  it 'returns the name of the team with the fewest tackles for a specific season' do
    expect(@stat_calculator.fewest_tackles("20132014")).to eq("Atlanta United")
    expect(@stat_calculator.fewest_tackles("20142015")).to eq("Orlando City SC")
  end

  it 'returns nil if there are no games for the season' do
    expect(@stat_calculator.fewest_tackles("99999999")).to be_nil
  end
end
```